docker-compose.test.yaml

```diff
@@ -1,6 +1,9 @@
-version: '3.4'
-services:
-  mine-support-api-gateway:
-    volumes:
-      - ./test-output:/usr/src/app/test-output
-    command: npm run test

```

Comment:
why remove the command? Doing so means it has to be set explicitly in the test script and the pipeline step when the `docker-compose.test.yaml`

Reply:
Good point. I went through a series of iterations where it made sense to remove the test compose file altogether but eventually came round to adding it back in. I find it quite odd to use `up` rather than `run` for this purpose but putting the npm command in one place is a good benefit.

Reply:
It turns out `run` doesn't require a command to be specified anyway, it falls back to the one in the compose file or the Dockerfile.